federation-rs-integration/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "federation-rs-integration",
+  "version": "1.0.0",
+  "description": "",
+  "main": "shell.ts",
+  "scripts": {
+    "clean": "rm -rf ./federation-rs ./node_modules"
```

Comment:
I _think_ the top level clean script should take care of this if the gitignore stuff is correct